contrib/http_servlet/build.gradle

```diff
@@ -10,12 +10,14 @@ description = 'OpenCensus Http Servlet Plugin'
 }
 
 def opencensusVersion = "0.18.0-SNAPSHOT"
```

Comment:
shouldn't this be 0.19.0-SNAPSHOT?

Reply:
I think you should do the same as you did for http util and change every 0.18.0 to 0.19.0 (including since)

Reply:
the above version is not used. I'll open another PR and remove it. There are no other since 0.18.0

Reply:
there is however reference to 0.18.0 in readme. I'll fix it.